README.md

```diff
@@ -27,3 +27,46 @@ Inside your module you can now watch for template changes :
 npm run watch  
 
 ```
+
+## Usage Grunt
+Add the dependency to your project using npm:
+
+    npm install --save-dev fh-wfm-template-build
+
+Load the plugin by adding it to your Gruntfile.js:
+
+    grunt.loadNpmTasks('fh-wfm-template-build');
+
+Add a configuration to your Gruntfile.js :
+
+    grunt.initConfig({
+      wfm_build: {
+        options: {
+          module: "wfm.<module-name>.directives",
+        }
+      },
+      wfm_watch: {
+        options: {
+          module: "wfm.<module-name>.directives",
+        }
+      },
```

Comment:
Its interesting if we can use some @Injection here and replace all directives with values